.github/workflows/test.yaml

```diff
@@ -0,0 +1,95 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      pytest_addopts:
+        description:
+          Extra options for pytest; use -vv for full details; see
+          https://docs.pytest.org/en/latest/example/simple.html#how-to-change-command-line-options-defaults
+        required: false
```

Comment:
Do you think this is the time to implement https://github.com/Tecnativa/docker-socket-proxy/issues/33#issuecomment-734840668?

I'm developing for TT25794 a new python lib that will help know that you're currently in the latest tag. I'll notify you when it's done.

Reply:
Yes, good suggestion. I agree that it makes sense now. I'll wait for your call then.